libs/design/switch/src/switch/switch.component.ts

```diff
@@ -81,12 +107,12 @@ export class DaffSwitchComponent {
 
   @Input() id: string = 'daff-switch-' + switchUniqueId++;
 
-  @Output() toggle = new EventEmitter<boolean>();
+  @Output() toggleSwitch = new EventEmitter<boolean>();
```

Comment:
I kinda liked `toggle`, do you prefer toggleSwitch?

Reply:
I was getting this error `Output bindings, including aliases, should not be named as standard DOM events`

Reply:
Ok, perhaps `switch` then? Thoughts @xelaint?

Reply:
`toggled`?